Lib/doctest.py

```diff
@@ -105,6 +105,8 @@ def _test():
 from io import StringIO, IncrementalNewlineDecoder
 from collections import namedtuple
 
+import _colorize  # Used in doctests
```

Comment:
Perhaps a somewhat redundant comment ;) it's the `doctest` module, afterall

```suggestion
import _colorize
```

Also, any reason this import is in a separate block from the other imports above it? Everything's the stdlib here ;)

Reply:
I followed the pattern of this, I think the comment is because it can show up as unused to tools:

https://github.com/python/cpython/blob/62aeb0ee69b06091396398de56dcb755ca3b9dc9/Lib/test/test_doctest/test_doctest.py#L7

But can remove if you prefer? And remove the existing one for `FakeInput`?

Reply:
Ahhh I misunderstood. The comment can stay, thanks for explaining!